rules/windows/builtin/security/win_security_susp_outbound_kerberos_connection.yml

```diff
@@ -17,13 +20,16 @@ detection:
     selection:
         EventID: 5156
         DestPort: 88
-    filter:
-        Application|endswith:
-            - '\lsass.exe'
-            - '\opera.exe'
-            - '\chrome.exe'
-            - '\firefox.exe'
-    condition: selection and not filter
+    filter_exact:
+        Application:
+            - 'C:\Windows\System32\lsass.exe'
+            - 'C:\Program Files\Google\Chrome\Application\chrome.exe'
+            - 'C:\Program Files\Mozilla Firefox\firefox.exe'
+    #filter_browsers:
+        #Application|endswith:
+        #    - '\opera.exe'
+        #    - '\tomcat\bin\tomcat8.exe'
```

Comment:
we should at least filter the full default paths.

Reply:
I don't have the paths for tomcat nor opera. It could be program files but I don't want to invest time in an FP that I don't know. I'll add them once I understand if these FPs are common as the rule says. They were added in older times without any comments. I need to recheck how and why they are generated.

Reply:
I don't like using stuff we added from feedback as we already get so little of it. But also stuff changes and it might no longer be necessary. I like hardening the filters but on the other hand adding some default paths should be possible. I can add the filters tomorrow. Right now I don't have access to a Windows VM.

Reply:
On the other hand, yeah why not. The changes were old and the filter is very permissive as is.